### PR TITLE
RODE-56: Add chart releaser

### PR DESCRIPTION
**Pull request by jknight-liatrio**

### CODE REVIEW

**Comment by mrparkers** [comment]
Can you merge main into this branch?  I see a lot of changes that I think are already in main

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - master
```
Comment (mrparkers) [inline .github/workflows/release.yml:6]:
```suggestion
      - main
```

Comment (jknight-liatrio) [reply .github/workflows/release.yml:6]: Lol I'll never get used to that...

Comment (jknight-liatrio) [reply .github/workflows/release.yml]: Fixed.

**Review by mrparkers — APPROVED** [review]

**Pull request merged**